combat-trainer.lic

```diff
@@ -3642,16 +3646,20 @@ class GameState
   end
 
   def perform_analyze?(combo_type, expertise_requirement)
-    return false if combo_type != 'flame' && !Flags["ct-#{combo_type}-ready"]
+    return false if combo_type != 'flame' && !Flags["ct-#{combo_type}-ready"] && DRStats.barbarian?
     return false if DRSkill.getrank('Expertise') < expertise_requirement
```

Comment:
Does this check work? Does the `getrank` method return something that an equation can be formed on for a non-barb?

Reply:
I checked this on my moonmage, and drskills says his expertise is 0

Reply:
OK great, thank you.